dlib/dnn/layers.h

```diff
@@ -1533,6 +1563,14 @@ namespace dlib
         visit_layers(net, impl::visitor_bn_running_stats_window_size(new_window_size));
     }
 
+    template <typename net_type>
+    void set_all_bn_inputs_no_bias (
+        net_type& net
+    )
+    {
+        visit_layers(net, impl::visitor_bn_input_no_bias());
+    }
+
```

Comment:
This should probably go in core.h next to the other visitors that do this kind of thing.  Like right below `set_bias_learning_rate_multiplier` is reasonable.

Reply:
I wanted to put it there in the first place, but I just put it here because the template parameter is `layer_mode` is defined in this file. I guess that's also why you put here the `visitor_bn_running_stats_window_size` and the `set_all_bn_running_stats_window_sizes`.

Reply:
Oh right, I forgot about `set_all_bn_running_stats_window_sizes`. Yeah, putting it here seems good :+1: